### DISCUSSION TITLE
Standard Notes keeps stopping

### DISCUSSION THREAD

**Discussion by cparish17** [standardnotes/forum#2859] [General]
**Describe the bug**
moved to a new cell phone, same model, Motorola One 5G. Now the applicaiton won't start
**To Reproduce**
1. Try to open trhe app
2
**Expected behavior**
Access to the aplication
**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Device: [e.g. Motorola One 5g Ace
 - OS: Android 10

**Additional context
Tried uninstall and re-install.. same problem

**Comment by wanoof** [comment]
I moved to a new phone and experienced the same issue.
The following worked for me:
1. Uninstall the app.
2. Download the app from the Standard Notes web page (download the APK, NOT the Google Play option) to your device.
3. Reinstall the app by opening the downloaded file. You might get a warning about installing apps from untrusted sources. Accept the warning and install, but remember to go back and reverse this setting after installing the app. In my case, this was under settings - apps and notifications - special app access - install unknown apps.
